network/f5/bigip_gtm_datacenter.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: bigip_gtm_datacenter
+short_description: Manage Datacenter configuration in BIG-IP
+description:
+  - Manage BIG-IP data center configuration. A data center defines the location
+    where the physical network components reside, such as the server and link
+    objects that share the same subnet on the network. This module is able to
+    manipulate the data center definitions in a BIG-IP
+version_added: "2.2"
+options:
+  contact:
+    description:
+      - The name of the contact for the data center.
+    required: false
+    default: None
+  description:
+    description:
+      - The description of the data center.
+    required: false
+    default: None
+  enabled:
+    description:
+      - Whether the data center should be enabled. At least one of state and
```

Comment:
`Whether the data center should be enabled. At least one of C(state) and C(enabled) are required.`